tools/flat-storage/src/flat.rs

```diff
@@ -0,0 +1,272 @@
+/// Tools for modifying flat storage - should be used only for experimentation & debugging.
+use clap::Parser;
+use near_chain::{
+    flat_storage_creator::FlatStorageShardCreator, types::RuntimeAdapter, ChainStore,
+    ChainStoreAccess,
+};
+use near_epoch_manager::EpochManagerAdapter;
+use near_primitives::{state::ValueRef, trie_key::trie_key_parsers::parse_account_id_from_raw_key};
+use near_store::{
+    flat_state::store_helper::{self, get_flat_head, get_flat_storage_creation_status},
+    Mode, NodeStorage, ShardUId, Store, StoreOpener,
+};
+use nearcore::{load_config, NearConfig, NightshadeRuntime};
+use std::{path::PathBuf, sync::Arc, time::Duration};
+use tqdm::tqdm;
+
+#[derive(Parser)]
+pub struct FlatStorageCommand {
+    #[clap(subcommand)]
+    subcmd: SubCommand,
+}
+
+#[derive(Parser)]
+#[clap(subcommand_required = true, arg_required_else_help = true)]
+enum SubCommand {
+    /// View the current state of flat storage
+    View,
+
+    /// Reset the flat storage state (remove all the contents)
+    Reset(ResetCmd),
+
+    /// Init the flat storage state, by copying from trie
+    Init(InitCmd),
+
+    /// Verify flat storage state
```

Comment:
Maybe add a comment like "can take hours if latest state size occupies tens of GBs"?